website/static/js/onboarder.js

```diff
@@ -422,6 +422,8 @@ function OBUploaderViewModel(params) {
     self.messageClass = ko.observable('text-info');
     // The target node to upload to to
     self.target = ko.observable(null);
+    //Boolean to track of if upload was successful
```

Comment:
Oh wait, just some small language changes. Can you remove the extra `to` and the `if` in the comments?